src/main/java/org/ditang/relaxng/defaults/RelaxNGDefaultValues.java

```diff
@@ -113,10 +138,15 @@ List<Attribute> getDefaultAttributes(String elementLocalName,
      *      java.lang.String, java.lang.String, java.lang.String,
      *      java.lang.String)
      */
+    @Override
     public void defaultValue(String elementLocalName, String elementNamespace,
         String attributeLocalName, String attributeNamepsace, String value) {
       String key = getKey(elementLocalName, elementNamespace);
-      List<Attribute> list = defaults.computeIfAbsent(key, k -> new ArrayList<>());
+      List<Attribute> list = defaults.get(key);
```

Comment:
Doesn't this change just change the code from `computeIfAbsent` to doing the exact same thing "manually"?